src/main/java/uk/gov/companieshouse/controller/ExceptionHandlerController.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.companieshouse.controller;
+
+import org.slf4j.Logger;
+import org.springframework.http.HttpStatus;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import uk.gov.companieshouse.exception.ConflictException;
+import uk.gov.companieshouse.exception.UnauthorisedException;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@RestControllerAdvice
+public class ExceptionHandlerController {
```

Comment:
@mateuszc-kainos see error handling below

I've tried to suppress the error data being returned from the API, and instead log this instead. Using `info` instead of `error` for all non-500 errors